charts/elasticsearch/templates/nginx/nginx-es-configmap.yaml

```diff
@@ -43,7 +43,11 @@ data:
         location = /_search {
           # This combined with disabling explicit index searching downstream
           # prevents any deployment from being able to query any other indexes.
+          {{- if .Values.global.loggingSidecar.enabled }}
+          rewrite ^/(.*) /vector.$remote_user.*/$1 break;
+          {{- else }}
           rewrite ^/(.*) /fluentd.$remote_user.*/$1 break;
+          {{- end }}
```

Comment:
I'm a little concerned that this config turns vector and fluentd into exclusive features, where previously we had been open to the idea of having them both running. I don't think this is a deal breaker, but it's something we need to be aware of.

Reply:
currently the default is fluentd, in case loggingSidecar is enabled we will use vector. IF needed we can define what index to be configured

Reply:
@danielhoherd good point. I remember it came up in some discussion that we could build it in a way that supports both side car logging with vector as well as our current fluentd implementation.
Do you think it is feasible though given this finding?
cc: @cmarteepants @pgvishnuram

Reply:
confirmed with @cmarteepants earlier that we will rarely run into a scenario where we will need both - https://github.com/astronomer/issues/issues/4577#issuecomment-1115299246